### PR TITLE
spellcheck turbinia

### PR DESCRIPTION
**Pull request by jaegeral**
Smaller typo fixes

### CODE REVIEW

File: tools/update-gke-infra.sh [hunk 1]
```diff
@@ -155,9 +155,9 @@ function update_to_latest {
 
 function resize_cluster {
     echo "Resizing cluster $CLUSTER_NAME to $CLUSTER_SIZE nodes."
-    read -p 'WARNING: This will delete nodes as well as any associated data on the node. Do you wish to continue? (yes/no) ' ANS
+    read -p 'WARNING: This will delete nodes as well as any associated data on the node. Do you wish to continue? (yes/no) ' AND
```
Comment (aarontp) [inline tools/update-gke-infra.sh:158]: I think this one is intentional (it seems to be short for "answer").

Comment (aarontp) [reply tools/update-gke-infra.sh:158]: @jaegeral ping

Comment (jaegeral) [reply tools/update-gke-infra.sh]: done

**Comment by aarontp** [comment]
FYI, it looks like there are some conflicts for this PR.

**Review by aarontp — APPROVED** [review]
LGTM, thanks!

**Pull request merged**